src/base/static/components/form-fields/toggle-field.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from "react";
+
+const baseClass = "mapseed-toggle-field";
```

Comment:
nit: why is this its own variable? It's only used once - can't it just be a string literal on the `className` attribute?

Reply:
basically just for consistency with the other components-- when I started I thought it would be useful to have a single place to declare the base component class, but this turned out to not be very useful in the end actually